llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -1758,16 +1758,11 @@ static void insertSpills(const FrameDataInfo &FrameData, coro::Shape &Shape) {
         PtrValue = Builder.CreateAnd(PtrValue, Builder.CreateNot(AlignMask));
         return Builder.CreateIntToPtr(PtrValue, AI->getType());
       }
-      // If the type of GEP is not equal to the type of AllocaInst, it implies
-      // that the AllocaInst may be reused in the Frame slot of other
-      // AllocaInst. So We cast GEP to the AllocaInst here to re-use
-      // the Frame storage.
-      //
-      // Note: If we change the strategy dealing with alignment, we need to refine
-      // this casting.
+      // If the type of GEP is not equal to the type of AllocaInst, it means
+      // that the AllocaInst uses a different address space.
```

Comment:
I prefer to remain the original comments.

Reply:
Yeah! I agree with you after a second look.